internal/endtoend/testdata/ddl_create_procedure/postgresql/pgx/query.sql

```diff
@@ -1,6 +1,8 @@
--- name: Placeholder :exec
-SELECT 1;
+-- name: CallInsertData :exec
+CALL insert_data($1, $2);
```

Comment:
We should add another test query to make sure that sqlc.arg is working correctly

```
CALL insert_data(sqlc.arg('foo'), sqlc.arg('bar'))
```

Reply:
Good spot, that did indeed break until i added 
```
	case *ast.CallStmt:
		a.apply(n, "FuncCall", nil, n.FuncCall)
```
into internal/sql/astutils/rewrite.go as that was falling through to the default and panicing